Content.Server/NPC/HTN/HTNComponent.cs

```diff
@@ -32,10 +32,10 @@ public sealed partial class HTNComponent : NPCComponent
     public float PlanCooldown = 0.45f;
 
     /// <summary>
-    /// How much longer until we can try re-planning. This will happen even during update in case something changed.
+    /// Game time when we can next try re-planning.
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
-    public float PlanAccumulator = 0f;
+    public TimeSpan NextPlanTime;
```

Comment:
DataFields usually need `[AutoPausedField]` here and `[AutoGenerateComponentPause]` for the component. But I'm not certain in this case, since it is not a DataField.

Reply:
Yes, an `[AutoPausedField]` would be needed to have `NextPlanTime` be bumped correctly after being unpaused. Though, if this is not added, the worse that can happen is that all the NPCs decide to compute new plans right after being unpaused.